app/assets/javascripts/social-share-button.coffee

```diff
@@ -44,7 +44,7 @@ window.SocialShareButton =
       when "google_bookmark"
         SocialShareButton.openUrl("https://www.google.com/bookmarks/mark?op=edit&output=popup&bkmk=#{url}&title=#{title}")
       when "delicious"
-        SocialShareButton.openUrl("http://www.delicious.com/save?url=#{url}&title=#{title}&jump=yes&pic=#{img}")
+        SocialShareButton.openUrl("http://www.del.icio.us/save?url=#{url}&title=#{title}&jump=yes&pic=#{img}")
```

Comment:
https://del.icio.us/

---

And I can't beleive that delicious.com can not open now. This website will be die. Orz